index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8"name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+  
+    <nav id="navbar">
+      <a href="#projects">project</a>
+    </nav>
+
+    
+
+    <section id="welcome-section">
+      <h1>Hey I am Mimic</h1>
+    </section>
+
+    <section id="projects">
+      <p class="project-tile"></p>
+      <a href="#">These are some of my projects</a>
+    </section>
```

Comment:
- [ ] I think it better to use different class names and id names to make it way easier and less complex where you want to style them 
- [ ] _{optional}_  so far you did well but in that case, you can consider checking how to use [HTML&CSS best practices ](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)